.gitlab/lassen-jobs.yml

```diff
@@ -31,7 +31,7 @@ xl_16_1_1_7:
 
 xl_16_1_1_7_gcc_8_3_1:
   variables:
-    SPEC: "%xl@16.1.1.7 cxxflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1 cflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1"
+    SPEC: "%xl@16.1.1.7 cxxflags='--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1 -qthreaded -std=c++14 -O3 -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qhot -qpic -qsmp=omp -std=c++11' cflags=--gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1"
```

Comment:
@kab163 @davidbeckingsale Sorry this is so hack-tastic . . . I tried creating a "+gcc @xl" SPEC, but Spack couldn't handle that. If there's a better way to do this, please let me know.

Reply:
No problem, this is the right way to add custom flags.